### DISCUSSION TITLE
DeepL translator should have part of it

### DISCUSSION THREAD

**Discussion by anujkmehrotra** [dialect-app/dialect#210] [General]
HI,
I believe that DeepL translator as a source should be part of it beacuse it is more accurate, if you people do so ??

**Comment by K0-RR** [comment]
If I understand correctly you want DeepL translator to be added as a backend to Dialect. This is a duplicate of https://github.com/dialect-app/dialect/issues/47.

**Reply by anujkmehrotra** [reply]
Yes, please.

**Reply by K0-RR** [reply]
Look at the mentioned issue to see why this is currently not possible.

**Reply by anujkmehrotra** [reply]
I read issue #47. It looks that it is paid API. I wish that you could add it.
But overall, you made a nice traslation app.

**Reply by K0-RR** [reply]
I'm just a user like you, not associated with the Dialect app team.
https://github.com/dialect-app/dialect/graphs/contributors these are the people who made it or helped.

**Reply by anujkmehrotra** [reply]
okay then, thanks to contributors.
